Mathlib/CategoryTheory/Comma/Basic.lean

```diff
@@ -409,6 +410,54 @@ def post (L : A ⥤ T) (R : B ⥤ T) (F : T ⥤ C) : Comma L R ⥤ Comma (L ⋙
       right := f.right
       w := by simp only [Functor.comp_map, ← F.map_comp, f.w] }
 
+/-- The canonical functor from the product of two categories to the comma category of their
```

Comment:
This should be a split into a separate file and not included in `Basic`. 

See http://speed.lean-fro.org/mathlib4/run-detail/99a581a1-3912-4f3b-aeab-4f557e34d7f0 It almost triples the instructions and is used in a more targeted fashion that results of `Basic`.

Reply:
We could put it into a `Discrete.lean` file you mean?